daemon/daemon.go

```diff
@@ -253,11 +317,10 @@ func (daemon *Daemon) restore() error {
 					activeSandboxes[c.NetworkSettings.SandboxID] = options
 					mapLock.Unlock()
 				}
+			} else { // !(c.IsRunning() || c.IsPaused())
```

Comment:
why this comment ?

Reply:
While doing the modification so I could remember what was the context :sweat_smile:. I can remove it.